packages/apps-config/src/endpoints/productionRelayPolkadot.ts

```diff
@@ -79,6 +79,15 @@ export function createPolkadot (t: TFunction): EndpointOption {
           Dwellir: 'wss://astar-rpc.dwellir.com'
         }
       },
+      {
```

Comment:
These need to be alphabetical, so CI will fail. Just move it down to the correct position.

Reply:
Ok @jacogr  I assumed it was in the sequence of the paraid.